forest-example/datasets.py

```diff
@@ -24,7 +24,7 @@ def load_labels(self) -> pd.DataFrame:
             PROJECT_ROOT / DataPaths.RAW_LABELS / "hansen_labelled_data.csv"
         )
 
-        df = df.sample(n=1000, random_state=42)
```

Comment:
Curious as to why?

Reply:
Because there's way too many points, so this was a way to test a few

Reply:
the change here is to the `random_state` - not super important, just curious why you changed it

Reply:
Oh! I think this was to force new points to be exported rather than the 1000 that already existed in Google Cloud Storage when I initially tested